pipeline/pipeline_runner.go

```diff
@@ -215,7 +215,7 @@ func Run(config *PipelineConfig) {
 			outputsWg.Done()
 			continue
 		}
-		LogInfo.Println("Output started: ", name)
+		LogInfo.Println("Output started:", name)
```

Comment:
Println adds a space between each operand anyways - they were getting two spaces after the colon, which is what made them stand out against all the other startup messages.